### PR TITLE
fix multiple include

### PR DESCRIPTION
**Pull request by mchun**
When there are 2 or more include statements, if the current query does't contain keys in the first include field, then result will be returned before finding the rest of the include fields.

### CODE REVIEW

**Comment by flessard** [comment]
+1

File: src/RestQuery.js [hunk 1]
```diff
@@ -377,7 +377,11 @@ RestQuery.prototype.handleInclude = function() {
       this.include = this.include.slice(1);
       return this.handleInclude();
     });
+  }else if(this.include.length >0){
```
Comment (gfosco) [inline src/RestQuery.js]: Can you update the spacing here?

`} else if (this.include.length > 0) {`

**Comment by drew-gross** [comment]
Looks like a `.DS_Store` file snuck into your commit. Can you remove that, then squash into a single commit please?

**Comment by mchun** [comment]
I am sorry for that. This is my first pull request ever and I am thrilled to have an opportunity to contribute.

**Comment by drew-gross** [comment]
Cool, congrats on your first PR, and welcome to the open source community!

**Pull request merged**